src/plugins/vis_augmenter/public/constants.ts

```diff
@@ -11,3 +11,6 @@ export const EVENT_MARK_SIZE = 100;
 export const EVENT_MARK_SIZE_ENLARGED = 140;
 export const EVENT_MARK_SHAPE = 'triangle-up';
 export const EVENT_TIMELINE_HEIGHT = 25;
+export const PLUGIN_AUGMENTATION_ENABLE_SETTING = 'visualization:enablePluginAugmentation';
+export const PLUGIN_AUGMENTATION_MAX_OBJECTS_SETTING =
+  'visualization:enablePluginAugmentation.maxPluginObjects';
```

Comment:
Rather than having duplicate constants in `public/constants.ts` and `server/constants.ts`, you can put them in a single file under `common/` so it can be accessed by both.